### ISSUE TITLE
Request: Make Properties window respect dark theme

### ISSUE THREAD

**Issue by kristoferblack** [Perchik71/Creation-Kit-Platform-Extended#76]
The properties window which is almost always open/being used currently does not go into dark mode, while the rest of the interface does. Since this is such a frequently used window, it'd be great if that could be added. The only other window having a remainder of light mode is the Layers window but I think I understand there are some limitations to what can be done there. Thanks!

![image](https://github.com/user-attachments/assets/dd40e029-96a6-4779-850e-cd6cd89216c0)

**Comment by yak3d** [comment]
I can look at this.

**Comment by Perchik71** [comment]
@yak3d Yes, you correctly noticed that this is a PropertyGrid, I do not know how to color it. 
First, need to determine exactly what this window is PropertyGrid, perhaps there is a hint in GetClassName().
Visit [PropertyGrid-Made-Easy](https://www.codeproject.com/Articles/77957/Win32-SDK-PropertyGrid-Made-Easy) it is colored by the WM_NOTIFY message, maybe need OWNERDRAW style.
